app/src/main/res/layout/delete_history_button.xml

```diff
@@ -2,12 +2,16 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do we need a new layout level surrounding the button? Adding deeper layouts costs a bit of performance.

Reply:
It has to do with adding an alpha to a recyclerview child. If it isn't nested, the built-in recylerview animations prevent us from adding our own alpha on the button (used for disabled state).

Reply:
Oh, good to know. Thank you.